eZ/Publish/API/Repository/Tests/FieldType/RichTextIntegrationTest.php

```diff
@@ -618,6 +620,107 @@ public function testConvertRemoteObjectIdToObjectId($test, $expected)
         );
     }
 
+    /**
+     * @param string $xmlDocumentPath
+     * @dataProvider providerForTestCreateContentWithValidCustomTag
+     */
+    public function testCreateContentWithValidCustomTag($xmlDocumentPath)
+    {
+        $validXmlDocument = $this->createDocument($xmlDocumentPath);
+        $this->createContent(new RichTextValue($validXmlDocument));
+    }
+
+    /**
+     * Data provider for testCreateContentWithValidCustomTag.
+     *
+     * @return array
+     */
+    public function providerForTestCreateContentWithValidCustomTag()
+    {
+        $data = [];
+        $iterator = new \DirectoryIterator(__DIR__ . '/_fixtures/ezrichtext/custom_tags/valid');
```

Comment:
Nitpick: Missing import of `\DirectoryIterator`

Reply:
Amended in 07afb66.